silk-rules/src/main/scala/org/silkframework/rule/TransformRule.scala

```diff
@@ -73,7 +73,7 @@ sealed trait TransformRule extends Operator {
     def collectPaths(param: Input): Seq[TypedPath] = param match {
       case p: PathInput if p.path.operators.isEmpty => Seq()
       case PathInput(_, path: TypedPath) => Seq(path)
-      case PathInput(_, path: Path) => Seq(TypedPath(path, AutoDetectValueType, isAttribute = false))
+      case PathInput(_, path: Path) => Seq(TypedPath(path, UntypedValueType, isAttribute = false)) //TODO TypedPath change: changed to UntypeValueType, else the select on arbitrary typed TypedPaths with StringValueType would fail
```

Comment:
Makes sense to use untyped path here, since the input paths of the transform rule are path requests.

Reply:
still untyped